examples/stable_diffusion_v2/ldm/modules/train/optim.py

```diff
@@ -1,28 +1,16 @@
-# Copyright 2022 Huawei Technologies Co., Ltd
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-# http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-# ============================================================================
 """
  build optimizer for ms
 """
 from mindspore.nn.optim.adam import Adam, AdamWeightDecay
 
 
-def build_optimizer(model, opts, lr):
+def build_optimizer(model, optim, betas, weight_decay, lr):
```

Comment:
not all optim use betas? I'd go for **optim_kwargs

Reply:
Both optimizers implemented require parameter betas, and I think all the parameters should be used explicitly in train_*.py